OCM/OCM.swift

```diff
@@ -636,10 +636,10 @@ open class OCM: NSObject {
     /**
      Use it to reset the localStorage of WebView
      
-     - Since: 2.0.14
+     - Since: 2.2.2
      */
-    public func isResetLocalStorageWebView(reset: Bool) {
-        Config.resetLocalStorageWebView = reset
+    public func resetLocalStorageWebView() {
```

Comment:
I think that this method should be called: resetWebViewsLocalStorage(), what do u think @jerilyngigigo ?

Reply:
it's a good suggestion, but maybe like this: resetWebViewLocalStorage?